tests/integration/test_charm.py

```diff
@@ -4,10 +4,11 @@
 
 import logging
 import os
+from typing import List
```

Comment:
add tests for partial scale up and partial scale down (you have full scale down which is great! But a partial would also be nice)

Reply:
I agree Mia. Implemented on https://github.com/canonical/postgresql-k8s-operator/pull/9/commits/3651c6d458a88df4287e8c0d389ce2bcac9fec76.